README.md

```diff
@@ -5,40 +5,60 @@
 
 A VS Code extension for Open Liberty. The extension will detect your Liberty Maven or Liberty Gradle project if it detects the `io.openliberty.tools:liberty-maven-plugin` in the `pom.xml` or `io.openliberty.tools:liberty-gradle-plugin` in the `build.gradle`. Through the Liberty Dev Dashboard, you can start, stop, or interact with Liberty dev mode on all available [Liberty Maven](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev) or [Liberty Gradle](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md) projects in your workspace.
 
-![Open Liberty Tools Extension](images/OpenLibertyTools.png)
+![Open Liberty Tools Extension](images/open-liberty-tools.png)
 
 ## Quick Start
+
 - Install the extension
 - Open Liberty supported projects will appear in the Liberty Dev Dashboard on the side bar
 - Right-click a project in the Liberty Dev Dashboard to view the available commands
 
 ## Features
+
 - View supported `liberty-maven-plugin`(version `3.1` or higher) or `liberty-gradle-plugin`(version `3.0` or higher) projects in the workspace
 - Start/Stop dev mode
 - Start dev mode with custom parameters
 - Run tests
 - View unit and integration test reports
 
+## Commands
+
+| Command                      | Description                                                                                                                                                                                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Start                        | Starts dev mode.                                                                                                                                                                                                                                                                                                             |
+| Start…​                      | Starts dev mode with custom parameters. Supported parameters can be found in the documentation for the [dev goal of the Liberty Maven Plugin](https://github.com/OpenLiberty/ci.maven/blob/master/docs/dev.md#dev) and the [dev task of the Liberty Gradle Plugin](https://github.com/OpenLiberty/ci.gradle/blob/master/docs/libertyDev.md#command-line-parameters). |
```

Comment:
For Gradle, the task name should be libertyDev.  e.g. `[libertyDev task of the Liberty Gradle Plugin]`